### PR TITLE
Bookstore: initialise project with components

### PR DESCRIPTION
**Pull request by fbclh**
**Project requirements**

- Initialise React app.
- Create a directory for your components.
- Add and set two <Route>s and <Link>s for the app's navigation:
- Books - the default view: Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- Should have a form for adding a book (no functionality yet). Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
- Categories: Should display a button "Check status" only.
- Styling is not required at this point.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const Book = ({ category, title, author }) => (
+  <li className="book-li">
+    <div className="book-container">
+      <h3>{category}</h3>
+      <h4>{title}</h4>
+      <p>{author}</p>
+      <div>
+        {/* <button type="button">Add</button> */}
+        <button type="button">Remove</button>
+        {/* <button type="button">Edit</button> */}
```
Comment (rukundoeric) [inline src/components/Book.js:12]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/components/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksList = ({ books }) => (
+  <ul>
+    {books.map((book) => (
+      <Book
+        // category={book.category}
```
Comment (rukundoeric) [inline src/components/BooksList.js:8]: - [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @fbclh, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Book.js (hunk 1 above)
Comment (rloterh) [reply src/components/Book.js]: This issue is well resolved! ✔️

File: src/components/BooksList.js (hunk 2 above)
Comment (rloterh) [reply src/components/BooksList.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
Hi Fabio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**